edit/location.go

```diff
@@ -148,6 +148,8 @@ func locStart(ed *Editor) {
 	// Pinned directories are also blacklisted to prevent them from showing up
 	// twice.
 	black := convertListsToSet(ed.locHidden(), ed.locPinned())
+	pwd, _ := os.Getwd()
```

Comment:
Please handle the possible error.

If there was an error, you can simply not add `pwd` to the blacklist.